lib/mongoid_session_store/store.rb

```diff
@@ -9,7 +9,7 @@ class Store < ActionDispatch::Session::AbstractStore
   private
 
     def generate_sid
-      BSON::ObjectId.new.to_s
+      SecureRandom.hex(32)
```

Comment:
Knowing how hash functions are working - there can be collisions too if we will hash tome too. The only way to prevent it - add time as row value. But! It's randomness and it's possiblt that same millisecond there will be created two sessions with same random hash. Huh? :smile: